examples/cifar2hdf5.sh

```diff
@@ -4,5 +4,5 @@
 # Consider changing the paths to files and directories
 #
 python datasets/cifar2hdf5.py \
---output_file ~/datasets/hdf5/cifar10.hdf5 \
---download_dir ~/datasets/
+--output_file ~/Storage/Datasets/cifar10/hdf5/cifar10.hdf5 \
```

Comment:
I'd leave the original paths in the public repository, which are meant to be generic. These scripts should serve as "examples", not as the go-to executable files.